ReactNative/ReactTwitter/core/views/main-navigator.js

```diff
@@ -64,28 +68,15 @@ var NavigationBarRouteMapper = {
   },
 
   _getTitleByScreenId (screenId) {
-    switch (screenId) {
-      case splashScreenID:
-        return 'SplashScreen'
-      case loginScreenID:
-        return 'Login'
-      case debugScreenID:
-        return 'Debug Screen'
-      case deepLinkingID:
-        return 'Deep Linking'
-      case pushNotificationsScreenID:
-        return 'Push Notifications'
-      case OauthViewID:
-        return 'Oauth'
-      case tweetsListID:
-        return 'Timeline'
-      case tweetViewID:
-        return 'Tweet Detail'
-      case debugAnimationSceneID:
-        return 'Debug Animation'
-      default:
-        return ''
-    }
+    let title = ''
+    scenes.forEach((scene) => {
+      if (scene.navigatorID() === screenId) {
+        title = scene.navigatorTitle()
+        return
```

Comment:
This does not work as you'd expect it would: all you get from this `return` is to exit the current iteration, which you still get if this is the last line 😄 

If you want to break of the loop, you need to use a `for ... in` and `return` from there.